pkg/cmd/cli/cmd/set/volume.go

```diff
@@ -387,6 +401,9 @@ func (v *VolumeOptions) Complete(f *clientcmd.Factory, cmd *cobra.Command, out,
 		}
 		v.AddOpts.ClaimSize = q.String()
 	}
+	if len(v.AddOpts.DefaultMode) == 0 {
+		v.AddOpts.DefaultMode = "644"
```

Comment:
Rather than handling the default value here, you can include it in your flag declaration and skip this block altogether:

```go
cmd.Flags().StringVarP(&addOpts.DefaultMode, "default-mode", "", "644", "The default mode bits to create files with. Can be between 0000 and 0777. Defaults to 0644.")
```